Dashboard/Dashboard/Presentation/Elements/CourseCardView.swift

```diff
@@ -84,16 +93,26 @@ struct CourseCardView: View {
     
     private var courseTitle: some View {
         VStack(alignment: .leading, spacing: 3) {
-            if let courseEndDate {
-                Text(courseEndDate.dateToString(style: .courseEndsMonthDDYear))
-                    .font(Theme.Fonts.labelSmall)
-                    .foregroundStyle(Theme.Colors.textSecondaryLight)
-                    .multilineTextAlignment(.leading)
-            } else if let courseStartDate {
-                Text(courseStartDate.dateToString(style: .courseStartsMonthDDYear))
-                    .font(Theme.Fonts.labelSmall)
-                    .foregroundStyle(Theme.Colors.textSecondaryLight)
-                    .multilineTextAlignment(.leading)
+//            if let courseEndDate {
```

Comment:
Are you sure that we don't need that code?

Reply:
`CourseAccessMessageView` is doing the job now. Removed the commented code.